### PR TITLE
[DDW-901] Newsfeed

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR implements `Newsfeed` feature.

## Todo

- [x] Activate SSL on daedalus.io domain and update Daedalus code to execute HTTPS request to fetch newsfeed JSON file
- [x] Introduce platform as array of available platforms for each item
- [x] Prevent Alert / Incident overlays before "Terms of use" and "Language Selection"
- [x] Ensure that automatic/manual update overlay is not blocked by Alert / Incident overlays
- [x] Implement JSON file verification

## Screenshots

Screenshot 2019-09-18 at 21 00 45
Screenshot 2019-09-18 at 20 58 34
Screenshot 2019-09-18 at 21 00 57
Screenshot 2019-09-18 at 21 01 13
Screenshot 2019-09-18 at 21 01 21

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test

_News definitions for different platforms and environments:_

**1. STAGING**
- You should see incident with title: "Incident 1 in English" or "Incident 1 in Japanese" on all platforms `macOS`, `Windows` and `Linux`

**2. TESTNET**
- You should see 3 alerts on all platforms `macOS`, `Windows` and `Linux`

**3. MAINNET**
- You should see 2 infos and 2 announcements on `macOS` & `Windows` platform
- `Linux` has one more entry and you should see 3 infos and 3 announcements
---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by gnpf** [comment]
@tomothespian For the unread announcement with the longer title, the dot is displayed on a different row.

image

**Comment by tomothespian** [comment]
> @tomothespian For the unread announcement with the longer title, the dot is displayed on a different row.
> 
> image

@a-rukin do you have some suggestion how we can improve that?

**Comment by a-rukin** [comment]
> > @tomothespian For the unread announcement with the longer title, the dot is displayed on a different row.
> > image
> 
> @a-rukin do you have some suggestion how we can improve that?

Can we apply some kind of "no-break" here? 
So if the dot goes next line, one word goes there also

**Comment by nikolaglumac** [comment]
PR looks good 👍 
All tests are green:
Screenshot 2019-09-26 at 13 28 57

File: source/renderer/app/api/utils/patchAdaApi.js [hunk 1]
```diff
@@ -185,4 +187,28 @@ export default (api: AdaApi) => {
   api.setNetworkBlockHeight = async (height: number) => {
     NETWORK_BLOCK_HEIGHT = height;
   };
+
+  api.setFakeNewsFeedJsonForTesting = (json: ?GetNewsResponse) => {
+    FAKE_NEWSFEED_JSON = json;
```
Comment (daniloprates) [inline source/renderer/app/api/utils/patchAdaApi.js]: shouldn't we change this variable name?

Comment (nikolaglumac) [reply source/renderer/app/api/utils/patchAdaApi.js]: @daniloprates you mean `json` one?

Comment (nikolaglumac) [reply source/renderer/app/api/utils/patchAdaApi.js]: @daniloprates I will change it to `fakeNewsfeedJson`

**Comment by nikolaglumac** [comment]
@daniloprates I have implemented requested changes. Can you please review again?

**Pull request merged**